task.py

```diff
@@ -18,16 +20,24 @@ def add_new_data_row_to_csv_file(filepath, row):
     tables.write_table_to_csv(data, filepath)
 
 
-def minimal_task():
-    items.load_work_item_from_environment()
-    local_data_filepath = items.get_work_item_file(workitem_filename)
-    with open(local_data_filepath, "r") as fin:
+def print_file_contents(filepath):
+    with open(filepath, "r") as fin:
+        print(f"Contents of {filepath}:")
         print(fin.read())
+
+def minimal_task():
+    items.get_input_work_item()
+    # POSSIBLE BUG: if you don't give it a new name, when you save the work item
+    #  even if you only make changes to the local version of data.csv
+    #  it will update the work item version of data.csv
+    local_data_filepath = items.get_work_item_file(workitem_filename,'local_data.csv')
```

Comment:
Maybe because you override the input file specified by the work item which is at root dir, so it's a common source of truth.

```suggestion
    # POSSIBLE BUG: if you don't give it a new name, when you save the work item
    #  even if you only make changes to the local version of data.csv
    #  it will update the work item version of data.csv
    local_data_filepath = items.get_work_item_file(workitem_filename, Path('output') / 'data.csv')
```

Reply:
fixed